tests/test_tokenization_common.py

```diff
@@ -4164,3 +4164,34 @@ def test_special_token_addition(self):
                         replace_additional_special_tokens=False,
                     )
                     self.assertEqual(tokenizer_2.additional_special_tokens, ["<other>", "<another>", "<tok>"])
+
+    def test_oov_token_ids(self):
+        # Test out-of-vocabulary token ids
+
+        if not self.test_slow_tokenizer:
+            return
+
+        vocab_size = len(self.get_tokenizer())  # == base vocab_size + len(added_vocabs)
+        oov_ids = set(range(vocab_size, vocab_size + 10))
+        non_oov_ids = list(range(vocab_size, vocab_size - 10, -1))
+
+        # Behaviors for tokenizers below should be consistent
+        tokenizer1 = self.get_tokenizer()
+        tokenizer2 = self.get_tokenizer(oov_error="replace")
+        tokenizer3 = self.get_rust_tokenizer() if self.test_rust_tokenizer else ...
```

Comment:
```suggestion
         if self.test_rust_tokenizer:
                  tokenizer3 = self.get_rust_tokenizer()
```
let's go for explicit versions here and below 😉